paper/paper.Rmd

```diff
@@ -23,15 +23,24 @@ knitr::opts_chunk$set(echo = TRUE)
 
-The R[@r-base] package partition is a fast and flexible data reduction tool that implements the Partition framework. partition enforces a minimum level of information, specified by the user, that reduced features must capture. Each feature begins as an independent cluster. Then, a bottom-up (agglomerative) approach grows clusters as much as possible, subject to the information loss constraint. Then, partition summarizes each feature in the subset into a single new feature. The reduced features are highly interpretable: original features map to one and only one feature in the reduced data set. The partition software is flexible, as well: how features are agglomerated, how information loss is measured, and the way data are reduced can all be customized. Additionally, partition has been thoroughly benchmarked and profiled, with key components written in C++ to improve performance.
+The package partition is a fast and flexible data reduction tool that implements the Partition framework for the statistical programming language R [@r-base]. partition enforces a minimum level of information, specified by the user, that reduced features must capture. Each feature begins as an independent cluster. Then, a bottom-up (agglomerative) approach grows clusters as much as possible, subject to the information loss constraint. Then, partition summarizes each feature in the subset into a single new feature. The reduced features are highly interpretable: original features map to one and only one feature in the reduced data set. The partition software is flexible, as well: how features are agglomerated, how information loss is measured, and the way data are reduced can all be customized. Additionally, we have thoroughly benchmarked and profiled partition, with critical components written in C++ to improve performance.
```

Comment:
Suggested changes starting at sentence 5:
Each feature subset is then summarized into a single new feature. The reduced features are highly interpretable because original features map to one and only one feature in the reduced data set. The partition software is flexible and customizable in the way features are agglomerated, information is measured, and data are reduced. Additionally, we have thoroughly benchmarked and profiled partition, with critical components written in C++ to improve performance.